apps/explorer/lib/explorer/chain.ex

```diff
@@ -3549,20 +3549,21 @@ defmodule Explorer.Chain do
         }
   def recent_collated_forward_transfers_for_rap(options \\ []) when is_list(options) do
     necessity_by_association = Keyword.get(options, :necessity_by_association, %{})
-    paging_options = Keyword.get(options, :paging_options, @default_paging_options)
+    paging_options =
+      Keyword.get(options, :paging_options, @default_paging_options)
 
     total_forward_transfers_count = forward_transfers_count()
 
     fetched_forward_transfers =
-        fetch_recent_collated_forward_transfers_for_rap(paging_options, necessity_by_association)
+        fetch_recent_collated_forward_transfers_for_rap(paging_options, necessity_by_association, true)
 
     %{total_forward_transfers_count: total_forward_transfers_count, forward_transfers: fetched_forward_transfers}
   end
 
-  def fetch_recent_collated_forward_transfers_for_rap(paging_options, necessity_by_association) do
+  defp fetch_recent_collated_forward_transfers_for_rap(paging_options, necessity_by_association, is_extra_transfer) do
```

Comment:
The new parameter isn't used in this method, but just passed into another method. This is typically not good practice and can be difficult to maintain down the line.